taskfiles/k8s.yml

```diff
@@ -140,7 +125,6 @@ tasks:
       - task: set-aio-max
         vars:
           USE_SUDO: "false"
-      - build-operator-images
```

Comment:
Does this get run elsewhere?

Reply:
Yes, https://github.com/redpanda-data/redpanda-operator/blob/5c31c49cd460bfe382458cd314ca36830cca1bf8/taskfiles/k8s.yml#L151